components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -36,6 +36,8 @@ constexpr char kRampID[] = "8yxja8782as5essk2myz3bmh4az6gpq4nte9n2gf";
 constexpr char kSardineStorefrontBaseURL[] = "https://crypto.sardine.ai";
 constexpr char kSardineClientTokensURL[] =
     "https://api.sardine.ai/v1/auth/client-tokens";
+constexpr char kTransferoURL[] =
+    "https://app-onramp-development.azurewebsites.net";
```

Comment:
This looks like the incorrect URL for this

Reply:
@kdenhartog This url will change once Transfero complete their URL migration which currently ongoing. This is the main reason this PR is in draft.

Reply:
All good, that's what I figured.